_sass/layouts/home.scss

```diff
@@ -5,7 +5,8 @@
 .homepage {
 
   background-image: url($baseurl + '/assets/images/bg-home-primary.png');
-  background-repeat: no-repeat;
+  background-repeat: repeat-x;
```

Comment:
Wouldn't `round` be better? Do you fear distortion?

Reply:
Not sure how Round would be a better solution. We have no need to repeat the image vertically and I've made the image seamlessly repeat on the X axis (while also making is smaller). This solves the issue for large screens and doesn't negatively impact mobile screens.